install_config/router/f5_router.adoc

```diff
@@ -53,6 +53,7 @@ requirements:
 * For edge integration (not recommended):
 ** A working ramp node
 ** A working tunnel to the ramp node
+* An {product-title} user assigned the xref:../../admin_guide/manage_scc.html#listing-security-context-constraints[`privileged` security context constraint]
```

Comment:
we always assign `privileged` scc to a service account like `router`, not an user.